doc/api.md

```diff
@@ -1,9 +1,13 @@
 # Publishing API's API
 
-This is the primary interface from publishing apps to the publishing pipeline. Applications PUT items as JSON conforming to a schema specified in govuk-content-schemas.
-Content paths are arbitrated internally by the Publishing API, the content is then forwarded to the live and draft content stores, and placed on the message queue for other apps (eg email-alert-service) to consume.
-
-## Index
+This is the primary interface from publishing apps to the publishing pipeline.
+Applications PUT items as JSON conforming to a schema specified in
+[govuk-content-schemas](https://github.com/alphagov/govuk-content-schemas).
+Content paths are arbitrated internally by the Publishing API, the content is
```

Comment:
Hmm I feel like that might veer this off into needing a longer introduction whilst it's good to keep it succinct. I think the use of "paths" does throw this off a little. Might be good with just "Content is arbitrated..."